plugins/copr.py

```diff
@@ -502,29 +515,34 @@ def _runtime_deps_warning(self, copr_username, copr_projectname):
         if not runtime_deps:
             return
 
-        print(_("Maintainer of the enabled Copr repository decided to make\n"
-                "it dependent on other repositories. Such repositories are\n"
-                "usually necessary for successful installation of RPMs from\n"
-                "the main Copr repository (runtime dependencies provided).\n\n"
-
-                "Be aware that the note about quality and bug-reporting\n"
-                "above applies here too, Fedora Project doesn't control the\n"
-                "content. Please review the list:"))
-        dep_idx = 1
-        for repo_id in runtime_deps:
-            print("\n{0}. [{1}]".format(dep_idx, repo_id))
-            print("   baseurl={0}".format(repo.cfg.getValue(repo_id, "baseurl")))
-            dep_idx += 1
-
-        msg = _("\nThese repositories have been enabled automatically."
-                "\nDo you want to keep them enabled?")
-        if self.base._promptWanted():
-            if self.base.conf.assumeno or not self.base.output.userconfirm(
-                    msg='{} [Y/n]: '.format(msg), defaultyes_msg='{} [Y/n]: '.format(msg)):
```

Comment:
This code here has a capital Y: `"[Y/n]"` in both `msg` and `defaultyes_msg` and your new code unifies it to `"[y/N]"` for `msg`, but that is you actually fixing a mistake, unless I'm missing something, right?

I just don't understand why the `userconfirm()` method doesn't abstract away the whole `"[y/N]"` prompt part...

Reply:
Correct.  I initially wanted to have n/Y by default in the second warning,
but then I was over-voted in previous PR... we changed the semantics,
but not the message.

Reply:
Well, no ... we don't have powers to decide the default in plugin, right?  It depends on the system configuration, and on my box it is e.g. always `y/N`.  Is that correct?

Reply:
> Well, no ... we don't have powers to decide the default in plugin, right? It depends on the system configuration, and on my box it is e.g. always `y/N`. Is that correct?

I understood it the same way.